Cocktail/Cocktail/ViewControllers/CocktailDetailViewController.swift

```diff
@@ -171,8 +171,25 @@ class CocktailDetailViewController: UIViewController {
         recipeGuideLabel.text = "Recipe".localized
         myTipGuideLabel.text = "Tip".localized
         ingredientsGuideLabel.text = "Ingredients".localized
+        
+        likeButton.addAction(UIAction(handler: { [weak self]_ in
+            guard let self = self, let cocktailData = self.cocktailData else { return }
+            if self.cocktailData?.wishList == true {
+                self.likeButton.setImage(UIImage(systemName: "heart"), for: .normal)
+                var modifiedRecipe = cocktailData
+                modifiedRecipe.wishList = false
+                updateRecipe(originRecipe: cocktailData, modifiedRecipe: modifiedRecipe, origin: self.originRecipe)
+            } else {
+                self.likeButton.setImage(UIImage(systemName: "heart.fill"), for: .normal)
+                var modifiedRecipe = cocktailData
+                modifiedRecipe.wishList = true
+                updateRecipe(originRecipe: cocktailData, modifiedRecipe: modifiedRecipe, origin: self.originRecipe)
+            }
```

Comment:
```suggestion
let hasWishList = self.cocktailData?.wishList == true
self.likeButton.setImage(UIImage(systemName: hasWishList ? "heart" : "heart.fill"), for: .normal)
var modifiedRecipe = cocktailData
modifiedRecipe.wishList = hasWishList ? false : true
updateRecipe(originRecipe: cocktailData, modifiedRecipe: modifiedRecipe, origin: self.originRecipe)
```
삼항조건연산자를 써서 이렇게 처리해도 깔끔할 듯
전체적인 로직은 동일하지만 조건에 따라서 일부 변수의 값이 달라지는 것이라면.